types/chart.js/index.d.ts

```diff
@@ -130,7 +130,7 @@ declare namespace Chart {
     }
 
     interface ChartData {
-        labels?: string[] | string[][];
+        labels?: (string | string[])[];
```

Comment:
My apologies, it looks like our tslint rules don't like complex array types using `[]`. You need to change the definition again to `labels?: Array<string | string[]>;` for the build to pass.